src/main/java/com/google/sps/data/Coordinates.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.data;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+
+/** 
+ * This class represents the coordinates of a site location.
+ */
+public class Coordinates {
+  @SerializedName(value = "Longitude")
+  private final double lng;
+
+  @SerializedName(value = "Latitude")
+  private final double lat;
```

Comment:
Also, I'm concerned that the use of final here will have unexpected consequences. Can you explain to need for these to be final, and the impact them being final will have for instances that are not built by reflection?

Reply:
I made it private final so that it wouldn't be edited or changed unknowingly by other methods just in case as a safety net, I will make it to public now